mlos_bench/mlos_bench/tests/optimizer/toy_optimization_loop_test.py

```diff
@@ -27,17 +28,20 @@ def _optimize(env: Environment, opt: Optimizer) -> Tuple[float, TunableGroups]:
         assert env.setup(tunables)
 
         (status, output) = env.run()
-        score = output['score']
         assert status.is_succeeded
+        assert output is not None
+        score = output['score']
         assert 60 <= score <= 120
 
         opt.register(tunables, status, score)
 
-    return opt.get_best_observation()
+    obvs = opt.get_best_observation()
+    assert isinstance(obvs, tuple) and len(obvs) == 2 and obvs[0] is not None and obvs[1] is not None
```

Comment:
Make sure we aren't returning `[None, None]` that the `get_best_observation()` could technically also return.